### PR TITLE
Updated menu command to be more efficient.

### PR DESCRIPTION
**Pull request by therealcs1010**
Did testing and refactored menu command. Along with minor bug fixing

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ before_deploy:
   - ./gradlew build
   - git config --local user.name "nwjbrandon"
   - git config --local user.email "nwjbrandon@gmail.com"
+  - export NODE_OPTIONS=–max_old_space_size=8192
```
Comment (modakshantanu) [inline .travis.yml]: :O

Comment (therealcs1010) [reply .travis.yml]: idk if this will work lol

Comment (therealcs1010) [reply .travis.yml]: Nope doesnt work

**Comment by therealcs1010** [comment]
image

**Pull request merged**